### ISSUE TITLE
SetWriteDeadline "not implemented" panic

### ISSUE THREAD

**Issue by wizofaus** [denisenkom/go-mssqldb#688]
Hi,
We're using an older version of this in an AWS lambda and just today we started getting "not implemented" panics when closing DB connections.  Nothing was changed in our code.
I can see these "not implemented" panics have been removed in the latest version, so is there any reason not to upgrade?
Would you have any clue why this would just start happening now (also, they don't happen on all our environments, but do happen in production, which is using RDS etc.)
It's true we deployed new binary builds of our lambdas recently, but we certainly didn't change any source code.
Thanks!

**Comment by tc-hib** [comment]
Have seen #639?

**Comment by wizofaus** [comment]
Actually I did, but just skimmed through it, on re-reading it in more detail it makes sense now - it's very likely the go compiler got updated on our build servers since our last version was built.
Just annoying we only spotted it in production (but thankfully it didn't result in a customer-facing error).

**Comment by berndverst** [comment]
Update your `go.mod` and make sure to use this version:

```
github.com/denisenkom/go-mssqldb v0.0.0-20210411162248-d9abbec934ba
```

**Comment by wizofaus** [comment]
Yeah have done so, seems fine.  Interestingly in AWS it only crashes when it's using RDS, vs an ec2 instance running SQL server (our non-profit set-up).

**Comment by berndverst** [comment]
@wizofaus FWIW it also crashed with Azure SQL Server -- couldn't quickly verify whether it crashed with the Docker images because they won't run on M1 Macs.
